src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -141,15 +143,18 @@ void TensorRTBuilder::AddLayer(int nid, const JSONGraphNode& node) {
     }
     params.inputs.push_back(input);
   }
-  ICHECK(converter->variable_input_count || converter->input_types.size() == params.inputs.size())
-      << "Op expected a different number of inputs.";
 
   // Convert op to TRT.
   converter->Convert(&params);
 
   // Get outputs.
   node_output_map_[nid] = {};
   for (auto out : params.outputs) {
+    auto out_type = params.inputs.at(1).weight.type == params.inputs.at(0).tensor->getType()
```

Comment:
Can you explain this? It seems very specific yet AddLayer is used for all  of the supported ops.

Reply:
This is unfortunately causing an vector index exception for me.